common-ui/src/main/res/layout/component_two_line_item.xml

```diff
@@ -151,13 +151,37 @@
         android:id="@+id/twoLineSwitchListItemWithDisabledSwitch"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:primaryText="Title"
-        app:secondaryText="Two line item with disabled switch"
+        app:primaryText="Two Line Item"
+        app:secondaryText="In disabled state"
         app:leadingIcon="@drawable/ic_globe_gray_16dp"
         app:showBetaPill="true"
         app:showSwitch="true"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/twoLineSwitchListItemWithPill"/>
 
+    <com.duckduckgo.mobile.android.ui.view.listitem.TwoLineListItem
+            android:id="@+id/twoLineSwitchListItemWithPrimaryTextColorOverlay"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:primaryText="Two Line Item"
+            app:secondaryText="With custom Primary Text color"
+            app:leadingIcon="@drawable/ic_globe_gray_16dp"
+            app:primaryTextColorOverlay="@color/red_text_color_selector"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/twoLineSwitchListItemWithDisabledSwitch"/>
+
+    <com.duckduckgo.mobile.android.ui.view.listitem.TwoLineListItem
+            android:id="@+id/twoLineSwitchListItemWithSecondaryTextColorOverlay"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:primaryText="Two Line Item"
+            app:secondaryText="With custom Secondary Text color"
+            app:leadingIcon="@drawable/ic_globe_gray_16dp"
+            app:secondaryTextColorOverlay="@color/red_text_color_selector"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/twoLineSwitchListItemWithPrimaryTextColorOverlay"/>
```

Comment:
NIT: You haven't added onClickListener for these new list items in the fragment. Maybe it was intentionally, so feel free to ignore it 🙂